_layouts/default.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
Very basic layout. We'll want something fancier that shows all the blog posts on a "home" page and a way to click into an individual post.